### PR TITLE
ADD flexsearch

### PR DESCRIPTION
**Pull request by ts-thomas**
# Pull request template for new library

Pull request for issue: #
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

## Profile of the lib

* Git repository (required): https://github.com/nextapps-de/flexsearch
* Official website (optional, not the repository):
* NPM package url (optional): https://www.npmjs.com/package/flexsearch
* License and its reference: Apache-2.0 https://github.com/nextapps-de/flexsearch/blob/master/LICENSE
* GitHub / Bitbucket popularity (required):
  * Count of watchers: 30
  * Count of stars: 2664
  * Count of forks: 45
* NPM download stats (optional):
  * Downloads in the last day: 
  * Downloads in the last week:
  * Downloads in the last month: 1767

## Essential checklist

* [x] I'm the author of this library
  * [x] I would like to add link to the page of this library on CDNJS on website / readme
* [x] This lib was not found on cdnjs repo
* [x] Doesn't already exist / has duplicate issue and PR
* [x] The lib has notable popularity
  * [x] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
  * [x] More than 800 downloads stats per month on npm registry
* [x] Project has public repository on famous online hosting platform (or been hosted on npm)

## Auto-update checklist

* [x] Has valid tags for each versions (for git auto-update)
* [x] Auto-update setup
* [x] Auto-update target/source is valid.
* [x] Auto-update filemap is correct.

## Git commit checklist

* [x] The first line of commit message is less then 50 chars; clean, clear and easy to understand.
* [x] The parent of the commit(s) in the PR is not older than 3 days.
* [ ] Pull request is sent from a non-master branch with a meaningful name.
* [x] Separate unrelated changes into different commits.
* [ ] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
* [ ] Close corresponding issue in commit message
* [ ] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/flexsearch/package.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "flexsearch",
+  "version": "0.3.61",
```
Comment (PeterDaveHello) [inline ajax/libs/flexsearch/package.json]: The version info needs to be removed as it's not added yet.

File: ajax/libs/flexsearch/package.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/nextapps-de/flexsearch.git"
+  },
+  "npmName": "flexsearch",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "dist/*.*"
```
Comment (PeterDaveHello) [inline ajax/libs/flexsearch/package.json]: and we can use `**/*` here

Comment (PeterDaveHello) [reply ajax/libs/flexsearch/package.json]: but I assume `flexsearch.node.js` is not needed?

**Comment by PeterDaveHello** [comment]
@ts-thomas would you please fill the pull request template about this library? Thanks.

**Comment by ts-thomas** [comment]
@PeterDaveHello Of course I will do. Thanks a lot!

**Pull request closed without merging**